src/redist/targets/packaging/windows/clisdk/variables.wxi

```diff
@@ -30,6 +30,6 @@
   <?error Invalid Platform ($(var.Platform))?>
   <?endif?>
 
-  <?define DependencyKey   = "Dotnet_CLI_$(var.SDKBundleVersion)_$(var.Platform)"?>
+  <?define DependencyKey   = "$(var.DependencyKeyPrefix)_$(var.SDKBundleVersion)_$(var.Platform)"?>
```

Comment:
Would `DependencyKeyName` be ok, to match Core-Setup?

Reply:
The 3 templates, the SdkPlaceholder, and the toolset MSIs were all sharing an identical Provider Key. In core-setup each MSI has a unique Provider Key - this is the preferable method.

Reply:
I might not understand your comment... my suggestion was to rename `DependencyKeyPrefix` to `DependencyKeyName` (sorry if that wasn't clear), I'm not sure how that relates to the values. That would remove a step to sharing the infra later with core-setup:

```xml
  <?define DependencyKey   = "$(var.DependencyKeyName)_$(var.BuildVersion)_$(var.Platform)"?>
```

Reply:
OIC - yes that would be great.

Reply:
Done.